### ISSUE TITLE
Trying to test but I'm failing. It's probably me.

### ISSUE THREAD

**Issue by kevinctofel** [scripting/twitter#1]
I was following the readme.md instructions in the test1 folder but not getting the expected result of the app reading a tweet and displaying its text. Granted, as a JS n00b, it's probably a user error. 

What I've done on a system with Node (v16.14.2) and npm (v8.5.0) installed:

- Cloned this repo. 
- Ran `node install` to pull in any dependencies/packages.
- Entered newly generated Twitter API keys in twitterConsumerKey and twitterConsumerSecret fields found in `config.json`
- Ran `node main.js`
- See what appears to be a server running on port 1491, per the attached image.

![Twitter test](https://user-images.githubusercontent.com/16373212/164353756-f7e129a7-81e9-496f-aa04-b53a9a854e47.jpg)

Not sure what to do next: enter a command in the server, navigate to a web page (presumably localhost: ####), or something else? 
I did nav to localhost:1491 and added some route info to the URL (/gettweet, for example) based on reading some of the code and got this console message: 
```
davetwitter/handleRequest: urlCallback == http://localhost/callbackFromTwitter?redirectUrl=undefined
returnError: jstruct == {
    "statusCode": 404,
    "data": "{\"errors\":[{\"message\":\"Sorry, that page does not exist\",\"code\":34}]}"
```

Any guidance is appreciated but if I'm in over my head as n00b, I won't be offended if you say so. ;)

**Comment by scripting** [comment]
This is a server app, as you discovered, and the readme says that too. So this is not what you want. I’ll have a look around to see if I have the code you want in the form you want.

also if you just want to try ideas out, you might try drummer which can do exactly what you want, and it’s as simple as possible.

**Comment by scripting** [comment]
Then I went and looked at test1. You have to load index.html in a web browser after launching the server app.

**Comment by kevinctofel** [comment]
Appreciate your checking and info, Dave. I tried loading index.html at localhost on port 1401 and on 1491, but the browser shows a "not found" message. Trying not to reinvent anything (as I likely couldn't anyway) and get some programmable read access to tweets. I'll close this out and look at Drummer, per your suggestion. Thanks!

**Comment by kevinctofel** [comment]
Closing, per the above comment.

**Comment by scripting** [comment]
Kevin, I tried running the test1 app too and got the same result. I'm not sure what is going wrong. 

It's an awful cycle -- I write this code hoping someone will try it, when I'm working on it, because it's best to catch problems when it's all fresh in my mind. But that never happens, so problems like this go unsolved. I probably haven't run that test code for years. And somewhere along the line it broke. 

I really want to encourage you to keep at it. You're a smart guy, and you got to the real problem in this. I need about ten guys like you to work with me on this stuff. I know it's not your responsibility, but it's still true. 

I'm not hearing much from Drummer users these days either, but I really tested the hell out of the Twitter verbs there, and documented them, look in the [DocServer site](http://docserver.scripting.com/?verb=twitter.addGroupToList) under Twitter. And if there are *any* problems please report them.

**Comment by kevinctofel** [comment]
No worries Dave, and I completely get where you're coming from. Once I move on from my little JS coding exercises to another one, a week goes by and if a classmate asks me about the older code, my mind goes blank. I can't imagine having to deal with that on a larger scale as you do. 

I will keep at it because I like solving puzzles and I like learning. And I did poke around in Drummer this morning, created a script to get Tweets, got it in my menu and it returned expected results. :)

**Comment by scripting** [comment]
@kevinctofel -- that's good!

BTW, the Drummer twitter verbs run through this code, there are just a couple of levels inbetween, factoring out the hard stuff. ;-)

**Comment by kevinctofel** [comment]
Right, that's what I gathered when browsing through the code files. I spent a good half hour looking around and while I won't claim to understand all of it, I get the general gist, features, and functions. If I don't have to spin up a server but could just use / call to Drummer for the Twitter bits I'm looking for, that will solve a big problem for a small project I have in mind.  I'll need to search though tweets for a specific keyword / regex, but I do that once I have the texts of the tweets. :)
